OrganisationSample/src/main/java/models/OrganisationDashBoard.java

```diff
@@ -0,0 +1,51 @@
+package models;
+
+import lombok.Data;
+import services.FieldGeneration;
+import services.FieldValidation;
+import utils.DepartmentNameUtility;
+import utils.EmployeeStatus;
+import utils.EmployeeTypeUtility;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Data
+public class OrganisationDashBoard {
+
+    private OrganisationEmployee organisationEmployee = new OrganisationEmployee();
```

Comment:
organisationEmployee should not be a member of this class, instead pass it to add employee method